server/multitenancy/routes.ts

```diff
@@ -52,7 +55,7 @@ export function setupMultitenantRoutes(
       cookie.tenant = tenant;
```

Comment:
Shall we encode in line 45 so that 55 and 58 will both get the encoded value?

Reply:
We can do that, while the most concern is to not producing HTML/script when responding to browser side, so encode at the last step is safer to me.